themes/openy_themes/openy_carnation/templates/elements/openy-class-sessions.html.twig

```diff
@@ -0,0 +1,80 @@
+<div class="container">
```

Comment:
can we have a dockblock, as requested by Drupal code style standard?

Reply:
@podarok Should we be adding this to the main modules template as well?

/modules/openy_features/openy_prgf/modules/openy_prgf_class_sessions/templates/openy-class-sessions.html.twig

Reply:
yes, please @aaronjchristian